### DISCUSSION TITLE
5.Ödevde Çıktı Hatası

### DISCUSSION THREAD

**Discussion by OmerK-Benkyzz** [gusanmaz/BMB212-Algorithms-2223#19] [Q&A] [answered]
5.Ödevde çıktının 8 olması gerekirken 1 yazmışsınız ama hocam kontrol edebilir misiniz. Desc bazında almamız gerekiyor olabilir mi

**Comment by gusanmaz** [comment, marked as answer]
Bir hata goremedim. 5. soruda GPA'yi `3.02` olan elemanin indis degeri istenmiyor. Kac adet GPA'yi `3.02` olan ogrenci oldugunun sayisinin tespit edilip ekran yazdirilmasi isteniyor. Ornek dosyada not ortalamasi `3.02` olan `1` ogrenci bulunuyor. Programinizin ekran yazdiracagi deger okuyacaginiz dosyaya gore 0 da olabilir, 1 de olabilir, 500 de olabilir.

Ornegin komut satiri argumani olarak https://github.com/gusanmaz/BMB212-Algorithms-22-23/blob/main/data/students/ordered/student_1K_des_gpa.csv dosyasinin yolunu alirsaniz programiniz `4` degerini dondurmeli.

Bu soruda Binary Search algoritmasinin klasiklesmis formunu degil bu soruna gore dusunup modifiye edeceginiz formunu kullanmaniz gerekiyor.
